lib/fix.js

```diff
@@ -0,0 +1,18 @@
+/** @babel */
+import lint from './lint';
+
+// (editor: Object) => function
```

Comment:
A habit I picked up when working on projects without type system. Remove or convert them to full-fledged JsDoc?